Source/Scene/GroundPrimitive.js

```diff
@@ -807,6 +903,19 @@ define([
         }
     }
 
+    GroundPrimitive._initialized = false;
+    GroundPrimitive.init = function() {
```

Comment:
`GroundPrimitive.initializeTerrainHeights` might be a better name for this.

Can we only require the user call this if they plan on using synchronous geometry?  i.e. if the primitive is async, can we just handle it ourselves?